examples/neural_compressor/launcher/PythonLauncher.md

```diff
@@ -0,0 +1,29 @@
+Use Python Launcher to Run Optimum-Intel Optimization with Intel Neural Compressor
+=====
+
+Users can use our Python **Launcher** design to run the Python model code as it is with automatic enabling of the Optimum-Intel optimization (by Intel Neural Compressor).
+
+## Quick-Start
+
+Example: Let's run an NLP model using ```run_glue.py``` from HuggingFace Transformers PyTorch [examples](https://github.com/huggingface/transformers/blob/v4.26-release/examples/pytorch/text-classification/run_glue.py).
```

Comment:
I would be in favor of adding tests before giving visiblity to `NeuralCoderAdaptor`

Reply:
OK, let me add the tests as well in this PR later.

Reply:
Test added, see comment below

Reply:
Thanks for adding it !